components/molecule/modal/src/index.scss

```diff
@@ -0,0 +1,108 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+@import './settings';
+
+body.is-MoleculeModal-open {
+  overflow-y: hidden;
+}
+
+.sui-MoleculeModal  {
+  align-items: center;
+  background-color: rgba($bg-molecule-modal, $op-molecule-modal);
+  bottom: 0;
+  display: none;
+  left: 0;
+  position: fixed;
+  right: 0;
+  text-align: center;
+  top: 0;
+  white-space: nowrap;
+  z-index: $z-molecule-modal;
+
+  &.is-MoleculeModal-open {
+    display: flex;
+    justify-content: center;
+    overflow-x: hidden;
+    overflow-y: auto;
+  }
+
+  &-dialog {
+    background-color: $bg-moleule-modal-dialog;
+    border: $bd-moleule-modal-dialog;
+    box-shadow: $bxsh-molecule-modal-dialog;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    margin: 0;
+    min-width: $miw-molecule-modal-dialog;
+    text-align: left;
+    vertical-align: middle;
+    white-space: normal;
+
+    @include media-breakpoint-up(xs) {
+      height: auto;
+      margin: $m-molecule-modal-dialog;
+      max-height: calc(100% - #{$m-molecule-modal-dialog}*2);
+      max-width: $maw-molecule-modal-dialog;
+    }
+
+    &--full {      
+      @include media-breakpoint-up(xs) {
+        max-width: $maw-molecule-modal-dialog--full;
+      }
+    }
+  }
+
+  &-header {
+    align-items: center;
+    background-color: $bg-molecule-modal-header;
+    border-bottom: $bd-molecule-modal-header;
+    color: $c-molecule-modal-header;
+    display: flex;
+    flex: 0 0 auto;
+    font-size: $fz-xl;
+    font-weight: $fw-regular;
+    justify-content: space-between;
+    padding: $pt-molecule-modal-header $pr-molecule-modal-header $pb-molecule-modal-header $pl-molecule-modal-header;
+    position: relative;
+  }
+
+  &-no-header {
+    align-items: center;
+    display: flex;
+    padding: 0;
+    position: relative;
+  }
+
+  &-close {
+    background-color: transparent;
+    border: 0;
+    cursor: pointer;
+    display: flex;
+    padding: $pt-molecule-modal-header $pr-molecule-modal-header $pb-molecule-modal-header $pl-molecule-modal-header;
```

Comment:
isn't there a conflict with the padding set by the header?

Reply:
I'm very confused by `paddings`